backend/app/core/security.py

```
@@ -25,3 +27,27 @@ def verify_password(plain_password: str, hashed_password: str) -> bool:
 
 def get_password_hash(password: str) -> str:
     return str(pwd_context.hash(password))
+
+
+def decrypt_password(encrypted_password_b64: str) -> str:
+    # 在测试环境中直接返回原始密码，跳过解密过程
+    if settings.TESTING or settings.TEST_MODE:
+        logger.debug("Testing mode: Skipping password decryption and returning the original password.")
```

Ruff (0.11.9)

Undefined name `logger`

(F821)